src/views/views.js

```diff
@@ -1,4 +1,4 @@
-import CreateChallenge from "./createChallenge/createChallenge";
-import EditChallenge from "./editChallenge/editChallenge";
-
+import CreateChallenge from "./challenges/createChallenge/createChallenge";
+import EditChallenge from "./challenges/editChallenge/editChallenge";
+// do we need this file?
```

Comment:
Yeah I would rather keep all views imported here and exported.

We could switch to the [aggregating export syntax](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#re-exporting_aggregating) though. But then let's do the same for `src/components` and `src/db`.

Reply:
Cool! I really like the aggregating export syntax. I've done it that way here for the `views` and created issues for the [src/db](https://github.com/bence-toth/challengify/issues/68) and [src/components](https://github.com/bence-toth/challengify/issues/69).